libs/core/src/lib/step-input/step-input.component.ts

```diff
@@ -490,7 +498,12 @@ export class StepInputComponent implements OnInit, AfterViewInit, OnDestroy, Con
             )
         );
 
-        return merge(onMouseDown$, onMouseDown$.pipe(switchMap(() => timerFactory$)));
+        return merge(
+            onMouseDown$,
+            onMouseDown$.pipe(switchMap(() => timerFactory$)),
+            // while key is pressed, event will be emitted continuously, so there's no need for timerFactory$
+            onKeyDownSpace$
```

Comment:
since the native button click can be triggered by Enter key let's add support for it as well

Reply:
done